CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v6.7 (2023-07-12)
```

Comment:
```suggestion
## Next release
```
We are not releasing this today, whenever we update the changelog, we typically just put `Next release` and then will have a separate PR to update the version and the date